### DISCUSSION TITLE
Release a ESM-only major version

### DISCUSSION THREAD

**Discussion by cossssmin** [posthtml/posthtml-components#100] [General]
@thewebartisan7 as part of an effort to migrate the PostHTML ecosystem to ESM, I'd like to work on a new major release for `posthtml-components` that would include:

- refactoring code to ESM-only
- require Node.js 18+
- migrate tests to use Vitest
- switch style linting to eslint
- add types

Please let me know if you're OK with this and I'll start work on the PR. Thanks!

**Comment by thewebartisan7** [comment]
Hey, sound goods to me! Let me know if you need my help with anything.

**Reply by cossssmin** [reply]
Thanks, will get started and open a PR and we can take it from there 👍
